### PR TITLE
[fs] Move getMountPoint from aos_communicationmanager to aos_common

### PR DESCRIPTION
**Pull request by MykolaSolyanko**

### CODE REVIEW

File: utils/fs/fs_test.go [hunk 1]
```diff
@@ -116,6 +116,68 @@ func TestMountAlreadyMounted(t *testing.T) {
 	}
 }
 
+func TestMountPoint(t *testing.T) {
+	if len(disk.Partitions) < 1 {
```
Comment (al1img) [inline utils/fs/fs_test.go:120]: Looks useless., setup fails if disk is not created.

File: utils/fs/fs_test.go [hunk 2]
```diff
@@ -184,6 +246,20 @@ func TestOverlayMount(t *testing.T) {
...
  **********************************************************************************************************************/
 
+func createDirAndGetMountPoint(mount string, dirName string) (mountPoint string, err error) {
+	firstDir := filepath.Join(mount, dirName)
+
+	if err = os.MkdirAll(firstDir, 0o755); err != nil {
+		return "", aoserrors.Wrap(err)
+	}
+
+	if mountPoint, err = fs.GetMountPoint(firstDir); err != nil {
```
Comment (al1img) [inline utils/fs/fs_test.go:256]: Do not put the function under test to another function. It reduces test readability (masks the function that is tested).
```go
os.MkdirAll

fs.GetMountPoint()
```
in test looks ok for me.

File: utils/fs/fs_test.go [hunk 3]
```diff
@@ -116,6 +116,68 @@ func TestMountAlreadyMounted(t *testing.T) {
...
+func TestMountPoint(t *testing.T) {
+	if len(disk.Partitions) < 1 {
+		t.Fatal("Count partitions must be at least 1")
+	}
+
+	type testData struct {
+		mountPointFirst  string
+		mountPointSecond string
+		compare          func(string, string) bool
+		skipUnmountError bool
```
Comment (al1img) [inline utils/fs/fs_test.go:128]: In test, compare first mount point and second mount point. If they are the same, umount only once.

File: utils/fs/fs_test.go (hunk 2 above)
Comment (MykolaSolyanko) [reply utils/fs/fs_test.go:255]: ok, got it

File: utils/fs/fs_test.go (hunk 3 above)
Comment (MykolaSolyanko) [reply utils/fs/fs_test.go]: Done

File: utils/fs/fs_test.go (hunk 2 above)
Comment (MykolaSolyanko) [reply utils/fs/fs_test.go:267]: Done

**Review by al1img — APPROVED** [review]

**Pull request merged**